dashboard/app/controllers/api_controller.rb

```diff
@@ -101,12 +102,18 @@ def user_progress_for_stage
           timestamp: last_activity.updated_at.to_datetime.to_milliseconds,
           source: level_source
         }
-
       end
       response[:disableSocialShare] = current_user.under_13?
       response[:disablePostMilestone] =
-        !Gatekeeper.allows('postMilestone', where: {script_name: script.name}, default: true)
+          !Gatekeeper.allows('postMilestone', where: {script_name: script.name}, default: true)
     end
+
+    slog(tag: 'activity_start',
+         script_level_id: script_level.id,
+         level_id: level.id,
+         user_agent: request.user_agent,
+         locale: locale) if level.finishable?
```

Comment:
That's right;  there is a single Ajax request to this API generated whenever a script level page is shown.

It has always been the case that a user might refresh the page multiple times resulting in multiple starts for a level followed by one finish, so I assume the activity monitor isn't disturbed by multiple start calls.  @Laurelfan or @joshlory can you confirm this is OK?